### ISSUE TITLE
problem downloading youtube video

### ISSUE THREAD

**Issue by GoogleCodeExporter** [MiraculousLegendas/get-flash-videos#454]
```
What steps will reproduce the problem?
1. Downloading youtube video

What is the expected output? What do you see instead?

$ get_flash_videos -y http://www.youtube.com/watch?v=IsrZt1Sv1kQ --debug enabled
No plugins installed
Downloading http://www.youtube.com/watch?v=IsrZt1Sv1kQ
-> GET http://www.youtube.com/watch?v=IsrZt1Sv1kQ
<- 200 text/html; charset=utf-8 (22263)
Trying to open plugin ~/.get_flash_videos/plugins/Www.pm
Using method 'youtube' for http://www.youtube.com/watch?v=IsrZt1Sv1kQ
get_youtube_video_info: 
http://www.youtube.com/get_video_info?&video_id=IsrZt1Sv1kQ&el=profilepage&ps=de
fault&eurl=http%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DIsrZt1Sv1kQ&hl=en_US&t=Il%
20corvo%201%20%281994%29%20film%20completo%20ITA%20%28The%20Crow%29
-> GET 
http://www.youtube.com/get_video_info?&video_id=IsrZt1Sv1kQ&el=profilepage&ps=de
fault&eurl=http%3A%2F%2Fwww.youtube.com%2Fwatch%3Fv%3DIsrZt1Sv1kQ&hl=en_US&t=Il%
20corvo%201%20%281994%29%20film%20completo%20ITA%20%28The%20Crow%29
<- 200 application/x-www-form-urlencoded (7621)
FlashVideo::Site::Youtube
$VAR1 = {
          'no_get_video_log' => '1',
          'status' => 'ok',
          'video_verticals' => '[34, 3]',
          'pltype' => 'contentugc',
          'ftoken' => '',
          'author' => 'LaCullaDelTrash',
          'vq' => 'auto',
          'allow_embed' => '1',
          'keywords' => 'film completo,b-movie,trash,cinema trash,horror,splatter,troma',
          'avg_rating' => '3.97674418605',
          'timestamp' => '1352931547',
          'video_id' => 'IsrZt1Sv1kQ',
          'view_count' => '30156',
          'allow_ratings' => '1',
          'plid' => 'AATOe-xPExx0bhrY',
          'token' => 'vjVQa1PpcFOW4o-4vZKya6M35Q2EM4Z0iORg5AQPIg8=',
          'watermark' => ',http://s.ytimg.com/yts/img/watermark/youtube_watermark-vflHX6b6E.png,http://s.ytimg.com/yts/img/watermark/youtube_hd_watermark-vflAzLcD6.png',
          'fexp' => '900304,923109,914003,907354,922401,920704,912806,927201,925706,922403,925003,913546,913556,916805,920201,901451',
          'url_encoded_fmt_stream_map' => 'itag=43&url=http%3A%2F%2Fo-o---preferred---sn-nx5cvox-hpa6---v7---lscache2.c.youtube.com%2Fvideoplayback%3Fupn%3D_fXvCBZBMxM%26sparams%3Dcp%252Cid%252Cip%252Cipbits%252Citag%252Cratebypass%252Csource%252Cupn%252Cexpire%26fexp%3D900304%252C923109%252C914003%252C907354%252C922401%252C920704%252C912806%252C927201%252C922403%252C925003%252C913546%252C913556%252C916805%252C920201%252C901451%26key%3Dyt1%26expire%3D1352956330%26itag%3D43%26ipbits%3D8%26sver%3D3%26ratebypass%3Dyes%26mt%3D1352931491%26ip%3D79.19.33.200%26mv%3Dm%26source%3Dyoutube%26ms%3Dau%26cp%3DU0hUR1lQVF9ISkNONF9RSlhEOl8xbFRJd0haYVJv%26id%3D22cad9b754afd644&type=video%2Fwebm%3B codecs%3D%22vp8.0%2C vorbis%22&fallback_host=tc.v7.cache2.c.youtube.com&sig=BDC38989158B6E297AAB5F7BA5E4825B595D92F1.B2EB06B67167C90F5442C757929CF70562F5E56A&quality=medium,itag=34&url=http%3A%2F%2Fo-o---preferred---sn-nx5cvox-hpar---v6---lscache7.c.youtube.com%2Fvideoplayback%3Fupn%3D_fXvCBZBMxM%26sparams%3Dalgorithm%252Cburst%252Ccp%252Cfactor%252Cid%252Cip%252Cipbits%252Citag%252Csource%252Cupn%252Cexpire%26fexp%3D900304%252C923109%252C914003%252C907354%252C922401%252C920704%252C912806%252C927201%252C922403%252C925003%252C913546%252C913556%252C916805%252C920201%252C901451%26expire%3D1352956330%26algorithm%3Dthrottle-factor%26burst%3D40%26ip%3D79.19.33.200%26itag%3D34%26sver%3D3%26key%3Dyt1%26mt%3D1352931491%26mv%3Dm%26source%3Dyoutube%26ms%3Dau%26ipbits%3D8%26factor%3D1.25%26cp%3DU0hUR1lQVF9ISkNONF9RSlhEOl8xbFRJd0haYVJv%26id%3D22cad9b754afd644&type=video%2Fx-flv&fallback_host=tc.v6.cache7.c.youtube.com&sig=282C2410E85A9408CECB9DD5D7202486BE968E85.03CD356E592B69FFFCBB1087275C2B1B9FBA1314&quality=medium,itag=18&url=http%3A%2F%2Fo-o---preferred---sn-nx5cvox-hpad---v8---lscache4.c.youtube.com%2Fvideoplayback%3Fupn%3D_fXvCBZBMxM%26sparams%3Dcp%252Cid%252Cip%252Cipbits%252Citag%252Cratebypass%252Csource%252Cupn%252Cexpire%26fexp%3D900304%252C923109%252C914003%252C907354%252C922401%252C920704%252C912806%252C927201%252C922403%252C925003%252C913546%252C913556%252C916805%252C920201%252C901451%26key%3Dyt1%26expire%3D1352956330%26itag%3D18%26ipbits%3D8%26sver%3D3%26ratebypass%3Dyes%26mt%3D1352931491%26ip%3D79.19.33.200%26mv%3Dm%26source%3Dyoutube%26ms%3Dau%26cp%3DU0hUR1lQVF9ISkNONF9RSlhEOl8xbFRJd0haYVJv%26id%3D22cad9b754afd644&type=video%2Fmp4%3B codecs%3D%22avc1.42001E%2C mp4a.40.2%22&fallback_host=tc.v8.cache4.c.youtube.com&sig=44364FECBE297F35DF0186E92A00AC0113B92265.75EF1B759C9B190395BB954BDF9D18FA8E40B690&quality=medium,itag=5&url=http%3A%2F%2Fo-o---preferred---sn-nx5cvox-hpar---v13---lscache7.c.youtube.com%2Fvideoplayback%3Fupn%3D_fXvCBZBMxM%26sparams%3Dalgorithm%252Cburst%252Ccp%252Cfactor%252Cid%252Cip%252Cipbits%252Citag%252Csource%252Cupn%252Cexpire%26fexp%3D900304%252C923109%252C914003%252C907354%252C922401%252C920704%252C912806%252C927201%252C922403%252C925003%252C913546%252C913556%252C916805%252C920201%252C901451%26expire%3D1352956330%26algorithm%3Dthrottle-factor%26burst%3D40%26ip%3D79.19.33.200%26itag%3D5%26sver%3D3%26key%3Dyt1%26mt%3D1352931491%26mv%3Dm%26source%3Dyoutube%26ms%3Dau%26ipbits%3D8%26factor%3D1.25%26cp%3DU0hUR1lQVF9ISkNONF9RSlhEOl8xbFRJd0haYVJv%26id%3D22cad9b754afd644&type=video%2Fx-flv&fallback_host=tc.v13.cache7.c.youtube.com&sig=BF7F3216F67DCAFBBB103C282C0B4CB18C75400E.5F92E4CF6D0BF95E4D6455BB45C91FFA897FA429&quality=small,itag=36&url=http%3A%2F%2Fo-o---preferred---sn-nx5cvox-hpad---v24---lscache4.c.youtube.com%2Fvideoplayback%3Fupn%3D_fXvCBZBMxM%26sparams%3Dalgorithm%252Cburst%252Ccp%252Cfactor%252Cid%252Cip%252Cipbits%252Citag%252Csource%252Cupn%252Cexpire%26fexp%3D900304%252C923109%252C914003%252C907354%252C922401%252C920704%252C912806%252C927201%252C922403%252C925003%252C913546%252C913556%252C916805%252C920201%252C901451%26expire%3D1352956330%26algorithm%3Dthrottle-factor%26burst%3D40%26ip%3D79.19.33.200%26itag%3D36%26sver%3D3%26key%3Dyt1%26mt%3D1352931491%26mv%3Dm%26source%3Dyoutube%26ms%3Dau%26ipbits%3D8%26factor%3D1.25%26cp%3DU0hUR1lQVF9ISkNONF9RSlhEOl8xbFRJd0haYVJv%26id%3D22cad9b754afd644&type=video%2F3gpp%3B codecs%3D%22mp4v.20.3%2C mp4a.40.2%22&fallback_host=tc.v24.cache4.c.youtube.com&sig=D7F2102BA63B203C154204FDB1D94CD2F46332A9.7CBC78596ED8DC4FF723CF447B003046E7B2BDD6&quality=small,itag=17&url=http%3A%2F%2Fo-o---preferred---sn-nx5cvox-hpar---v10---lscache7.c.youtube.com%2Fvideoplayback%3Fupn%3D_fXvCBZBMxM%26sparams%3Dalgorithm%252Cburst%252Ccp%252Cfactor%252Cid%252Cip%252Cipbits%252Citag%252Csource%252Cupn%252Cexpire%26fexp%3D900304%252C923109%252C914003%252C907354%252C922401%252C920704%252C912806%252C927201%252C922403%252C925003%252C913546%252C913556%252C916805%252C920201%252C901451%26expire%3D1352956330%26algorithm%3Dthrottle-factor%26burst%3D40%26ip%3D79.19.33.200%26itag%3D17%26sver%3D3%26key%3Dyt1%26mt%3D1352931491%26mv%3Dm%26source%3Dyoutube%26ms%3Dau%26ipbits%3D8%26factor%3D1.25%26cp%3DU0hUR1lQVF9ISkNONF9RSlhEOl8xbFRJd0haYVJv%26id%3D22cad9b754afd644&type=video%2F3gpp%3B codecs%3D%22mp4v.20.3%2C mp4a.40.2%22&fallback_host=tc.v10.cache7.c.youtube.com&sig=045B680EFD61D85FF680D7669E0099007543534C.AB99EB6A28771BE99814253BA13620A7560CDD76&quality=small',
          'ptk' => 'youtube_none',
          'sendtmp' => '1',
          'tmi' => '1',
          'account_playback_token' => '',
          'endscreen_module' => 'http://s.ytimg.com/yts/swfbin/endscreen-vfl88NFgt.swf',
          'thumbnail_url' => 'http://i2.ytimg.com/vi/IsrZt1Sv1kQ/default.jpg',
          'muted' => '0',
          'track_embed' => '0',
          'length_seconds' => '4880',
          'storyboard_spec' => 'http://i2.ytimg.com/sb/IsrZt1Sv1kQ/storyboard3_L$L/$N.jpg|48#27#100#10#10#0#default#_x1jKjgLfUFWybHA9G06F3pM4wE|60#45#489#10#10#10000#M$M#los1iMcbxRKX6HDJHXVdVZQe7zs|120#90#489#5#5#10000#M$M#vBpXsLmQX9jbr0_XItO52naeiYs',
          'fmt_list' => '43/640x360/99/0/0,34/640x360/9/0/115,18/640x360/9/0/115,5/320x240/7/0/0,36/320x240/99/0/0,17/176x144/99/0/0',
          'has_cc' => 'False',
          'title' => 'Il Bosco 1 (1988 - Andrea Marfori) - COMPLETO'
        };

Exists on http://www.youtube.com/get_video?fmt=37&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?fmt=22&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?fmt=35&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?fmt=34&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?fmt=18&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?fmt=5&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?fmt=13&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?fmt=17&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?fmt=38&video_id=IsrZt1Sv1kQ&t=Il 
corvo 1 (1994) film completo ITA (The Crow): 404
Exists on http://www.youtube.com/get_video?video_id=IsrZt1Sv1kQ&t=Il corvo 1 
(1994) film completo ITA (The Crow): 404
Error: Unable to find video URL

Couldn't extract Flash movie URL. This site may need specific support adding,
or fixing.

Please confirm the site is using Flash video and if you have Flash available
check that the URL really works(!).

Check for updates by running: /usr/bin/get_flash_videos --update

If the latest version does not support this please open a bug (or
contribute a patch!) at http://code.google.com/p/get-flash-videos/
make sure you include the output with --debug enabled.
Downloading http://enabled
-> GET http://enabled
<- 500 text/plain (146)
Couldn't download 'http://enabled': 500 Can't connect to enabled:80 (Bad 
hostname)
Trying to open plugin ~/.get_flash_videos/plugins/Enabled.pm
Using method 'generic' for http://enabled
-> GET 
<- 500 text/plain (146)
Error: Couldn't download URL: 500 Can't connect to enabled:80 (Bad hostname) at 
/usr/bin/get_flash_videos line 5980.

Couldn't extract Flash movie URL. This site may need specific support adding,
or fixing.

Please confirm the site is using Flash video and if you have Flash available
check that the URL really works(!).

Check for updates by running: /usr/bin/get_flash_videos --update

If the latest version does not support this please open a bug (or
contribute a patch!) at http://code.google.com/p/get-flash-videos/
make sure you include the output with --debug enabled.
Couldn't download any videos.

What version of the product are you using? On what operating system?

The last one (I updated) on Ubuntu 12.04 LTS


Please provide any additional information below.

```

Original issue reported on code.google.com by `biagioci...@gmail.com` on 14 Nov 2012 at 10:33

**Comment by GoogleCodeExporter** [comment]
```
This is with OpenBSD amd64 current, using get_flash_videos

$ get_flash_videos http://www.youtube.com/watch?v=IsrZt1Sv1kQ
Using method 'youtube' for http://www.youtube.com/watch?v=IsrZt1Sv1kQ
Downloading http://www.youtube.com/watch?v=IsrZt1Sv1kQ
Downloading 
http://o-o---preferred---sn-aig7kned---v6---lscache7.c.youtube.com/videoplayback
?upn=0_iyynTq_S4&sparams=algorithm%2Cburst%2Ccp%2Cfactor%2Cid%2Cip%2Cipbits%2Cit
ag%2Csource%2Cupn%2Cexpire&fexp=906717%2C906335%2C905296%2C922401%2C920704%2C912
806%2C927201%2C922403%2C925003%2C913546%2C913556%2C916805%2C920201%2C901451&expi
re=1352959930&algorithm=throttle-factor&burst=40&ip=80.177.13.158&itag=34&sver=3
&key=yt1&mt=1352936289&mv=m&source=youtube&ms=au&ipbits=8&factor=1.25&cp=U0hUR1l
QV19ISkNONF9RSkFKOjhrQ2NUSy0wbXlr&id=22cad9b754afd644&signature=3C2663C0DD34D855
9F111558F11C8289986FA406.220039C73D0E4E3A5F98C156E5D7E1947247495D...
Il_Bosco_1_(1988_-_Andrea_Marfori)_-_COMPLETO.mp4: 0% (1986.04 / 275896.23 
KiB)^C

See issue 453, that gives details of how you might update

Attached is an attempt to create a debian update, installs on Debian Squeeze - 
just working out how to do this, so may not work, and may not work on ubuntu. 
Download and do sudo dpkg -i get-flash-videos_1.24_2.deb
This could conflict with an existing package or not install because of the 
version numbering. A replacement package should come from the ubuntu package 
maintainer, and issue reported there first.
```

Original comment by `njtaylor...@gmail.com` on 15 Nov 2012 at 12:11

Attachments:
- [get-flash-videos_1.24-2_all.deb](https://storage.googleapis.com/google-code-attachments/get-flash-videos/issue-454/comment-1/get-flash-videos_1.24-2_all.deb)

**Comment by GoogleCodeExporter** [comment]
```
Thank you very much! I tried 1.25 version and 1.24-2_all but don't work both, 
then I solved with combined-get_flash_videos-1.24-git-20121106 (from issue 453) 
that works very well.
```

Original comment by `biagioci...@gmail.com` on 15 Nov 2012 at 1:25

**Comment by GoogleCodeExporter** [comment]
```
Issue resolved by updating version.

```

Original comment by `njtaylor...@gmail.com` on 31 Dec 2012 at 10:30
- Changed state: **Duplicate**
